### ISSUE TITLE
Security improvements suggestions

### ISSUE THREAD

**Issue by oleg31337** [mad-ady/tl-sg-prometheus-exporter#6]
1. It would be nice not to run the tool as root. This tool doesn't need elevated privilege, so systemd can run this as a regular user with no login capabilities.
2. It would be nice to only bind to localhost and not expose to entire network. start_http_server supports that if you supply '127.0.0.1' as a second argument. This could be a configuration parameter. The use case would be running tool on the same host as prometheus/influxdb

**Comment by mad-ady** [comment]
I agree! I was too lazy to create a new user for systemd, but it can be done.
However, I haven't had time to revisit the code, so it will either make it into a todo-list, or I am happy to accept a pull request.
